template/src/includes/settings/essentials.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+define('MAILGUN_API_KEY', '<%= mailgun %>');
```

Comment:
Hoe vaak gebruiken we nou mailgun? Is het echt zo moeilijk om dit zelf toe te voegen wanneer het nodig is?

Reply:
Te weinig, 9/10 keer als je een wordpress site start werkt mailen niet zonder mailgun plugin

Reply:
Kleine moeite om erin te houden toch?